modules/rosa-aws-provisioned.adoc

```diff
@@ -131,3 +131,11 @@ AWS security groups provide security at the protocol and port access level; they
 |`19531`
 
 |===
+
+[id="rosa-security-groups-custom_{context}"]
+=== Additional custom security groups
+When you create a cluster using an existing non-managed VPC, you can add additional custom security groups during cluster creation. Custom security groups are subject to the following limitations:
+
+* You must create the custom security groups in AWS before you create the cluster. For more information, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-security-groups.html[Amazon EC2 security groups for Linux instances].
+* You must associate the custom security groups with the VPC that the cluster will be installed into. Your custom security groups cannot be associated with another VPC.
+* You may need to request additional quota for your VPC if you are adding more than 2 additional custom security groups. For information on AWS quota requirements for ROSA, see _Required AWS service quotas_ in _Prepare your environment_. For information on requesting an AWS quota increase, see link:https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html[Requesting a quota increase].
```

Comment:
Is _additional quota_ the correct term? It sounds a bit odd to me. Also, I don't see that we use it elsewhere in the docs.